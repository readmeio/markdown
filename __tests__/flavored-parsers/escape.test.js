import { mdast } from '../../index';

describe('Escape', () => {
  it('uses the "escape" type', () => {
    const md = `\\&para;`;
    expect(mdast(md, { settings: { position: true } })).toMatchSnapshot();
  });
});
