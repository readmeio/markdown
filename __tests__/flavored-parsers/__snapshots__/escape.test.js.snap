// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Escape uses the "escape" type 1`] = `
Object {
  "children": Array [
    Object {
      "children": Array [
        Object {
          "position": Position {
            "end": Object {
              "column": 3,
              "line": 1,
              "offset": 2,
            },
            "indent": Array [],
            "start": Object {
              "column": 1,
              "line": 1,
              "offset": 0,
            },
          },
          "type": "escape",
          "value": "&",
        },
        Object {
          "position": Position {
            "end": Object {
              "column": 8,
              "line": 1,
              "offset": 7,
            },
            "indent": Array [],
            "start": Object {
              "column": 3,
              "line": 1,
              "offset": 2,
            },
          },
          "type": "text",
          "value": "para;",
        },
      ],
      "position": Position {
        "end": Object {
          "column": 8,
          "line": 1,
          "offset": 7,
        },
        "indent": Array [],
        "start": Object {
          "column": 1,
          "line": 1,
          "offset": 0,
        },
      },
      "type": "paragraph",
    },
  ],
  "position": Object {
    "end": Object {
      "column": 2,
      "line": 3,
      "offset": 10,
    },
    "start": Object {
      "column": 1,
      "line": 1,
      "offset": 0,
    },
  },
  "type": "root",
}
`;
